initfiles/componentfiles/configschema/xsd/dali.xsd

```diff
@@ -95,15 +95,15 @@
                             </xs:restriction>
                         </xs:simpleType>
                     </xs:attribute>
-                    <xs:attribute name="authMethod" type="xs:string" hpcc:presetValue="kerberos"
+                    <xs:attribute name="authMethod" type="xs:string" hpcc:presetValue=""
```

Comment:
Should the "authMethod" attribute be removed? Also LDAP/secmgrPlugin probably don't belong here...